packages/contracts/src/escrow/offchain.ts

```diff
@@ -71,7 +71,8 @@ export class MeshEscrowContract extends MeshTxInitiator {
     await this.mesh
       .txOut(scriptAddr, escrowAmount)
       .txOutInlineDatumValue(
-        initiateEscrowDatum(walletAddress, escrowAmount),
+        // TODO: remove dodgy cast
```

Comment:
Given that the only genuine code change within this PR (search for "This is the only code change within the PR.") exists only to deal with a strange default export issue, the fact that we're now having to perform some explicit casts potentially indicates that types weren't always being pulled through correctly even internally?